python/python/glide/async_commands/server_modules/ft.py

```diff
@@ -54,3 +54,25 @@ async def create(
         for field in schema:
             args.extend(field.toArgs())
     return cast(TOK, await client.custom_command(args))
+
+
+async def dropindex(client: TGlideClient, indexName: TEncodable):
+    """
+    Drops an index. The index definition and associated content are deleted. Keys are unaffected.
+
+    Args:
+        client (TGlideClient): The client to execute the command.
+        indexName (TEncodable): The index name for the index to be dropped.
+
+    Returns:
+        Returns a simple string "OK" message or an error reply.
+
+    Examples:
+        For the following example to work, an index named 'idx' must be already created. If not created, you will get an error.
+        >>> from glide.async_commands.server_modules import ft
+        >>> indexName = "idx"
+        >>> result = await ft.dropindex(glide_client, indexName)
+            b'OK'  # Indicates successful deletion/dropping of index named 'idx'
```

Comment:
ok is actually a string, need to fix that in all places, I will fix in json
we're so used to the bytes we forgot ok is simple


cc: @Yury-Fridlyand @acarbonetto

Reply:
b'OK'  # Indicates successful deletion/dropping of index named 'idx'
```suggestion
            'OK'  # Indicates successful deletion/dropping of index named 'idx'
```
Do you mean change it to this?